gender_novels/novel.py

```diff
@@ -142,6 +143,33 @@ def find_quoted_text(self):
 
         return quotes
 
+    def get_count(self, word):
+        """
+        Returns the number of instances of str word in the text.  N.B.: Not case-sensitive.
+        >>> from gender_novels import novel
+        >>> summary = ("Hester was convicted of adultery. ",
+        ...            "which made her very sad, and then Arthur was also sad, and everybody was ",
+        ...            "sad and then Arthur died and it was very sad.  Sadness.")
```

Comment:
The way you're initializing summary is not as a string but rather as a tuple of strings, so the tokenization function doesn't work properly. It's the difference between 
```
string = 'Two words'
tuple = ('Two', 'words')
```
You can fix that for example by using
```
summary = 'Hester was convicted of adultery.'
summary += 'which made her very sad,...'
```